### DISCUSSION TITLE
MCMC traceplot for lens_model_DESIJ2158%2B0257

### DISCUSSION THREAD

**Discussion by Rajit13** [AstroBridge/BDLensing#98] [Show and tell]
I'm posting this for the benefit of all. I am currently running and trying to improve the trace plot to better understand the convergence.
I already ran `2000` iterations. I am confused about what should be considered proper convergence. Can someone point out a parameter in the plot below which converged? I am counting 8/9 right now. Should I run more iterations? Disclaimer The `mcmc_backend.h5` file is 45 MB in size! 

![image](https://github.com/ajshajib/BDLensing/assets/81336262/be39225c-d303-4373-8eae-199c27c195ca)

**Comment by Rajit13** [comment]
**Update**: after `2800` steps I think all the parameters have converged. Can someone verify? 
![image](https://github.com/ajshajib/BDLensing/assets/81336262/9daca77d-55ed-4b05-9057-45f190f3b98b)

Also without ignoring the `burn` I got the below corner plot 
![image](https://github.com/ajshajib/BDLensing/assets/81336262/f3655439-0cf6-4d36-94d7-ac1228cd1eee)

**Reply by ajshajib** [reply]
Yes, very nice! The chain looks to have converged shortly after 2400 steps. So, you should make the corner plot only with the samples after that point.
